test/integration/targets/gcp_compute_http_health_check/tasks/main.yml

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
Not a blocker, but it would be good to have a test to check resource modification as well. That could apply for #35920 too.